custom-icons/a11y.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@radinamatic I minified the SVG file you provided because it contained a lot of Inkscape-specific information.